src/modules/roles/application/use-cases/__tests__/update-user-roles.use-case.spec.ts

```
@@ -275,7 +301,12 @@ describe('UpdateUserRolesUseCase', () => {
         roles: newRoles,
       });
 
-      const result = await useCase.execute(userId, undefined, roleIds, currentUserId);
+      const result = await useCase.execute(
```

ESLint

[error] 'result' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)